package.json

```diff
@@ -4,25 +4,24 @@
   "description": "Optimistic Nightfall",
   "main": "./nightfall-client/src/index.mjs",
   "scripts": {
-    "ping-pong": "mocha --timeout 0 --bail --exit test/ping-pong/ping-pong.test.mjs",
     "lint": "eslint . --ext js,mjs,jsx,ts,tsx && find-unused-exports",
     "prepare": "husky install",
     "doc:build:sdk": "jsdoc -c jsdoc.json cli/lib/nf3.mjs",
     "build-adversary": "node test/adversary/transpile-adversary.mjs",
-    "nightfall-down": "docker-compose -f docker/docker-compose.yml -f docker/docker-compose.dev.yml -f docker/docker-compose.ganache.yml -f docker/docker-compose.apps.yml -p 'nightfall_3' down -v",
```

Comment:
Why did you remove this script? It is being used in the pipeline.

Reply:
good catch! This should not have been removed 😅